enrichments/trace/internal/elastic/span.go

```diff
@@ -463,6 +463,9 @@ func (s *spanEnrichmentContext) setDestinationService(span ptrace.Span) {
 				s.serverAddress, s.serverPort, // fallback
 			); res != "" {
 				destnResource = res
+			} else {
+				// fallback to RPC service
+				destnResource = s.rpcService
```

Comment:
We already use `rpc.service` for setting `service.target.name`, so , it makes sense to me to use it for setting destination resource too.